Makefile

```diff
@@ -66,8 +66,37 @@ coveralls: $(COV_REPORT)
 
 .PHONY: bench
 BENCH ?= .
+BENCH_FILE ?= .bench/new.txt
 bench:
-	$(ECHO_V)$(foreach pkg,$(BENCH_PKGS),go test -bench=$(BENCH) -run="^$$" $(BENCH_FLAGS) $(pkg);)
+	$(ECHO_V)$(foreach pkg,$(BENCH_PKGS),go test -bench=. -run="^$$" $(BENCH_FLAGS) $(pkg) |\
+		tee $(BENCH_FILE);)
+
+.PHONY: benchbase
+BASELINE_BENCH_FILE=.bench/old.txt
```

Comment:
you usually define the PHONY right above the target. so swp this line with line 74